scripts/pypi_publish

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
+SCRIPTS_DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+SRC_DIR="$( cd -P "$( dirname "$SCRIPTS_DIR" )" && pwd )"
+
+plugins=("rastervision_pipeline" "rastervision_aws_batch" "rastervision_aws_s3" "rastervision_core" "rastervision_pytorch_learner" "rastervision_pytorch_backend" "rastervision_gdal_vsi")
+
+# Check if the --test flag is passed
+publish_to_test=false
+if [[ "$1" == "--test" ]]; then
+    publish_to_test=true
+    # Remove the --test flag from the arguments
+    shift
+fi
+
+# Function to publish a plugin
+publish_plugin() {
+    cd "$SRC_DIR/$1"
+    echo "Publishing $1 ... "
+    if [ "$publish_to_test" = true ]; then
+        twine upload --repository testpypi dist/*
+    else
+        twine upload dist/*
+    fi
+    echo "Done."
+    cd "$SRC_DIR"
+}
+
+# Publish each plugin
+for plugin in "${plugins[@]}"; do
+    publish_plugin "$plugin"
+done
+
```

Comment:
The only thing that I might suggest -- from a belt-and-suspenders point of view -- is to force a test publication before a real one.  If the person says passes `--test` then it will terminate after the test publication, if they do not then it will publish to testpypi then prompt them to inspect the published artifacts before proceeding with the real publication.

Reply:
To keep things simple, I've made it so that
- `--test` works as before
- With no `--test`, you have to go through a `Actually publish to PyPi? (y/N)` prompt
- `-y` bypasses the prompt (for when/if we choose to fully automate things)